lib/search.js

```diff
@@ -166,7 +166,7 @@ class Search {
 
 		let query = filterBuilder("anime", filter);
 
-		query += "id title { romaji english native userPreferred } } } }";
```

Comment:
We can already filter results by both format and status for anime and manga, see: [MediaFilterTypes](https://katsurin.com/docs/anilist-node/global.html#MediaFilterTypes) and the [filtering tutorial](https://katsurin.com/docs/anilist-node/tutorial-Filtering.html). I'm curious on a more elaborate reasoning behind this seeing as there is filtering already added.

Reply:
I don't want to filter, I need that info on searching.
![image](https://user-images.githubusercontent.com/69076992/211223963-d91c12b1-1da2-4685-a4c0-0e7d9bc99073.png)

Reply:
Ah, that makes a lot more sense. This is good then. I'm going to make a comment on the TypeScript definitions since that will need to be updated as well.